UnityPy/helpers/ArchiveStorageManager.py

```diff
@@ -101,6 +106,10 @@ def __init__(self, reader: EndianBinaryReader) -> None:
         )
 
     def decrypt_block(self, data: bytes, index: int):
+
+        if UnityPyBoost:
+            return UnityPyBoost.decrypt_block(bytes(self.index), bytes(self.substitute), data, index)
```

Comment:
The bytes cast for self.index and self.substitute is redundant as they are already bytes objects.
Given the buffer protocol it wouldn't even be an issue if they would be bytearrays